src/mixins/filing-template-mixin.ts

```diff
@@ -776,9 +778,9 @@ export default class FilingTemplateMixin extends DateMixin {
    * @param filing the restoration filing
    * @param entitySnapshot the latest entity snapshot
    */
-  parseRestorationFiling (filing: RestorationFilingIF, entitySnapshot: EntitySnapshotIF): void {
+  parseRestorationFiling (filing: RestorationFilingIF): void {
     // store Entity Snapshot
-    this.setEntitySnapshot(entitySnapshot)
+    const entitySnapshot = this.getEntitySnapshot
```

Comment:
This doesn't do what the comment says it does. What is the purpose of this code? Is it obsolete now due to some other changes?

Reply:
I'll be fixing the comment! We're now getting the entity snapshot here not storing it. It's being stored now in `limitedRestorationExtension.vue` 
For exact line:
https://github.com/JazzarKarim/business-edit-ui/blob/fc30d04a839eb4cfda728a136c1df4ca0d4f6cb3/src/views/LimitedRestorationExtension.vue#L240